nacho-bot/nachobot.js

```diff
@@ -43,4 +43,17 @@ client.on('message', message => {
     }
 });
 
+let now = new Date();
+let day;
+
+var interval = setInterval(function(){
+    now = new Date();
+    if (day != now.getDay()) {
+        scrape.scrape();
+        day = now.getDay();
+    } else {
+        console.log('Already Scraped Today!');
```

Comment:
this should be changed so it does not spam the output with already scraped indefinitely.